djangocms_versioning/static/djangocms_versioning/js/compare.js

```diff
@@ -0,0 +1,28 @@
+function bindHandlers(submitButton){
+    // Reloading the page to renew the csrf token which prevented multiple version compare tabs from opening
+    submitButton.addEventListener("click", function (e) {
+        window.location.reload()
+    });
+}
+
+window.addEventListener('load', function() {
+  document.querySelector('select[name=action]').addEventListener('change', function() {
+    // Always set handlersBound to false when the select option changed
+    let handlersBound = false;
+    let selectVal = this.options[this.selectedIndex].getAttribute("value");
+    const actionsForm = document.getElementById('changelist-form')
+    const submitButton = actionsForm.querySelector('[type="submit"]')
+
+    if (selectVal == "compare_versions") {
+        if (handlersBound === false) {
+            // Setting the target to blank to ensure the compare view is opened in a new tab
+            actionsForm.setAttribute('target', '_blank');
+            bindHandlers(submitButton);
+            handlersBound = true
+        }
+    }else {
+        // If the user deselect the compare version action, the "target=_blank" is removed
+        actionsForm.removeAttribute('target');
```

Comment:
This will cause an issue where the user selects another action which removes the target attribute, then when they clock the button the target has been undone. The set attribute in this case has to be added everytime the compare versions is selected but not inside the is bound check. 

Apart form that everything else is good.